src/pf-notification-drawer/pf-notification-drawer.spec.js

```diff
@@ -0,0 +1,141 @@
+describe ('PatternFly Notification Drawer Component Tests', function () {
+  let drawer, btn;
+
+  function render(html) {
+    let template = document.createElement('template');
+    template.innerHTML = html;
+    document.body.appendChild(document.importNode(template.content, true));
+  }
+
+  beforeEach(function (done) {
+    let mo = new MutationObserver(function () {
```

Comment:
Again, I'm not sure what our stand is regarding the use of MutationObserver. We seem to be using them in more than a few places.